cmd/api/auth/authz_test.go

```diff
@@ -5,9 +5,9 @@ package auth
 
 import (
 	"context"
-	"fmt"
 	"github.com/gin-gonic/gin"
 	"github.com/stretchr/testify/assert"
+	authnv1 "k8s.io/api/authentication/v1"
```

Comment:
```suggestion
	apiAuthn "k8s.io/api/authentication/v1"
```

I think you can drop the "v1" from this abbreviation. I doubt we will be using v1 and v2 at the same time. But, you should add "api" to make it clearer what package this is.

Reply:
Changed to apiAuthnv1. See previous thread for more info.